README.md

```diff
@@ -108,7 +104,48 @@ MTKAssertAlertIsPresented(
     ]
 )
 ```
- 
+
+### UIBarButtonItem
+
+Verify that a bar button item has the expected target/action pair and that the target actually responds to the selector that will be sent to it. 
+
+```swift
+MTKAssertBarButtonItem(testVC.editBarButtonItem, sends: #selector(MyViewController.didTapEditButton(_:)), to: testVC) 
+```
+
+### UIControl
+
+With a single assertion, you can verify that your control actions are hooked up and that your target actually responds to the selector that will be sent to it. 
+
+```swift
+MTKAssertControl(testVC.loginButton, sends: #selector(LoginViewController.didTapLoginButton(_:)), to: testVC, for: .touchUpInside, "The login button should be hooked up to the login action.") 
+```
+
+
+### UICollectionViewCell
+
+Using a `UICollectionView` extension, you can easily retrieve cells using the `testCell` method.  This method assures that the collection view returns a cell for the given index path, and that it is of the type expected.  Once these checks have been made, the cell is passed into the completion handler in order to execute further tests.
+
+```swift
+testVC.collectionView.testCell(at: indexPath) { (testCell: MyCollectionViewCell) in 
```

Comment:
Coupled with my comment above about reordering the parameters and adding the `as` function label, my personal preference would be to suggest the following usage, as it reads more fluently and doesn't rely as much on generic inference to determine the expected cell type.

```swift
testVC.collectionView.testCell(at: indexPath, as: MyCustomCell.self) { testCell in 
    XCTAssertEqual(testCell.label.text, "Hello World!")
}
```